.travis.yml

```diff
@@ -43,7 +44,7 @@ env:
   - TARGET=SPRACINGF3
   - TARGET=SPRACINGF3EVO
 #  - TARGET=SPRACINGF3MINI
-  - TARGET=STM32F3DISCOVERY
```

Comment:
Why are we doing this? I thought `STM32F3DISCOVERY` was the target that had intentionally all hardware features enabled, so we'd get maximum coverage and testing for interdependencies, so building this as part of the CI suite would make a lot of sense.